packages/@react-spectrum/card/docs/CardView.mdx

```diff
@@ -0,0 +1,522 @@
+
+```tsx example
+// Using the same CardView as above
+<PokemonCardView selectionMode="single" />
+```
+
+### Disallow empty selection
+CardView also supports a `disallowEmptySelection` prop which forces the user to have at least one Card in the CardView selected at all times.
+In this mode, if a single card is selected and the user presses it, it will not be deselected.
+
+```tsx example
+// Using the same CardView as above
+<PokemonCardView disallowEmptySelection />
+```
+
+### Disabled cards
+You can disable specific card by providing an array of keys to CardView via the `disabledKeys` prop. This will prevent cards from being selectable as shown in the example below.
+
+```tsx example
+// Using the same CardView as above
+<PokemonCardView selectionMode="multiple" disabledKeys={[3]} />
+```
+
+### Card actions
+A common use case is to allow a user to perform actions on one or more cards. The example below illustrates how one could use ActionBar to trigger an action on the
+currently selected card.
+
+```tsx example
+import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
+import Copy from '@spectrum-icons/workflow/Copy';
+import Delete from '@spectrum-icons/workflow/Delete';
+import Edit from '@spectrum-icons/workflow/Edit';
+
+let items = [
+  // Same items as in the pokemon examples...
+  ///- begin collapse -///
+  {id: 1, name: 'Bulbasaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/001.png'},
+  {id: 2, name: 'Ivysaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/002.png'},
+  {id: 3, name: 'Venusaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/003.png'},
+  {id: 4, name: 'Charmander', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/004.png'},
+  {id: 5, name: 'Charmeleon', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/005.png'},
+  {id: 6, name: 'Charizard', type: 'Fire, Flying', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/006.png'}
+  ///- end collapse -///
+];
+
+function CardViewActions() {
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+
+  return (
+    <ActionBarContainer height="500px">
+      <CardView items={items} selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} height="500px" maxWidth="800px" layout={GridLayout} aria-label="CardView Pokemon with ActionBar example" selectionMode="single">
+        {item => (
+          <Card textValue={item.name} width={item.width} height={item.height}>
+            <Image src={item.src} />
+            <Heading>{item.name}</Heading>
+            <Text slot="detail">PNG</Text>
+            <Content>{item.type}</Content>
+          </Card>
+        )}
+      </CardView>
+      <ActionBar
```

Comment:
I think due to the maxWidth on the CardView this looks off center. Maybe we should also make it use the blue style of action bar since that is the one most people should use